# Patch by Vlastimil Babka <caster@gentoo.org>
# Straightforward compatilibity with java6 interface extensions
--- src/java/org/apache/log/output/db/DefaultDataSource.java.orig	2009-11-06 18:03:16.000000000 +0100
+++ src/java/org/apache/log/output/db/DefaultDataSource.java	2009-11-06 18:04:05.000000000 +0100
@@ -149,4 +149,12 @@
     {
         m_logWriter = logWriter;
     }
+
+    public boolean isWrapperFor(Class cls) {
+        return false;
+    }
+ 
+    public Object unwrap(Class cls) throws SQLException {
+        throw new SQLException("Not a wrapper for " + cls);
+    }
 }
